lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,175 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone',
+        'js/views/fields',
+        'js/student_account/models/user_account_model',
+        'js/student_account/models/user_preferences_model',
+        'js/student_account/views/account_settings_fields',
+        'js/student_account/views/account_settings_view',
+    ], function (gettext, $, _, Backbone, FieldViews, UserAccountModel, UserPreferencesModel,
+                 AccountSettingsFieldViews, AccountSettingsView) {
+
+        return function (fieldsData, userAccountsApiUrl, userPreferencesApiUrl) {
+
+            var accountSettingsElement = $('.wrapper-account-settings');
+
+            var userAccountModel = new UserAccountModel();
+            userAccountModel.url = userAccountsApiUrl;
+
+            var userPreferencesModel = new UserPreferencesModel();
+            userPreferencesModel.url = userPreferencesApiUrl;
+
+            var sectionsData = [
+                 {
+                    title: gettext('Basic Account Information (required)'),
+                    fields: [
+                        {
+                            view: new FieldViews.ReadonlyFieldView({
+                                model: userAccountModel,
+                                title: gettext('Username'),
+                                valueAttribute: 'username',
+                                helpMessage: ''
+                            })
+                        },
+                        {
+                            view: new FieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('Full Name'),
+                                valueAttribute: 'name',
+                                helpMessage: gettext('The name that appears on your edX certificates.')
```

Comment:
Change help text to:

The name that appears on your edX certificates. Other learners never see your full name.